packages/ie11-polyfills/README.md

```diff
@@ -0,0 +1,13 @@
+# @fluentui/ie11-polyfills
+
+**IE11 CSS variables polyfill**
+
+Package to hold ie11 polyfill that allows webpages that use css variables for styling to render correctly in ie11. It will also hold any future ie11 polyfills
+
+To import ie11-polyfill
+
+```js
+import '@fluentui/ie11-polyfills';
+```
+
+It just works. If you are using any websites with hot reloading, you will have to either put the polyfill on each page that uses css variables, or refresh the page for the script to take effect.
```

Comment:
Some nitpicky suggestions
```suggestion
Package to hold IE11 polyfill that allows webpages that use CSS variables for styling to render correctly in IE11. It will also hold any future IE11 polyfills

To use the polyfills, just import the package:

```js
import '@fluentui/ie11-polyfills';
```

If you are using any websites with hot reloading, you will have to either put the polyfill on each page that uses CSS variables, or refresh the page for the script to take effect.
```